spark/src/main/scala/geotrellis/spark/tiling/ZoomedLayoutScheme.scala

```diff
@@ -116,15 +116,23 @@ class ZoomedLayoutScheme(val crs: CRS, val tileSize: Int, resolutionThreshold: D
 
   def zoomOut(level: LayoutLevel) = {
     val layout = level.layout
+
+    if(layout.layoutCols < 2 || layout.layoutRows < 2)
+      sys.error(
```

Comment:
This is too late to give this exception. At this point a couple of pyramid levels would have already been saved and all that work would have been wasted.

Its possible to give this exception before we start the pyramid but now the nature of this error depends on work in the broken out PR. I would say lets remove this exception here and deal with the issue fully in separate PR.